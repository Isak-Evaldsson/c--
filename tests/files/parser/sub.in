int a(){
    int a = 1 - 5 + 3;
}