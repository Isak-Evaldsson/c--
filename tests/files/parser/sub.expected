Prototype: a
    VarDecl: a
        BinOpExpr: +
            BinOpExpr: -
                LiteralExpr: 1
                LiteralExpr: 5
            LiteralExpr: 3
